### PR TITLE
enable build of performance-addon-operator metadata

### PR DESCRIPTION
**Pull request by ffromani**
We have a new pure metadata image.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml [hunk 1]
```diff
@@ -36,6 +36,8 @@ images:
 - dockerfile_path: openshift-ci/Dockerfile.must-gather
   from: os-min8
   to: performance-addon-operator-must-gather
+- dockerfile_path: openshift-ci/Dockerfile.metadata
```
Comment (yanirq) [inline ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml:39]: change to Dockerfile.bundle

Comment (fromanirh) [reply ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml]: done in a later change

**Pull request merged**